Danki2/Assets/Scenes/SampleScene.unity

```diff
@@ -3114,10 +3114,24 @@ PrefabInstance:
       propertyPath: m_LocalEulerAnglesHint.z
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 1191838369, guid: 79ac0796178cdea4bb5cbf3704b54d4a, type: 3}
+      propertyPath: m_IsKinematic
+      value: 0
+      objectReference: {fileID: 0}
     - target: {fileID: -5201030127241623248, guid: 79ac0796178cdea4bb5cbf3704b54d4a,
         type: 3}
       propertyPath: baseStats._values.Array.data[1]
-      value: 7
+      value: 6
+      objectReference: {fileID: 0}
+    - target: {fileID: -5201030127241623248, guid: 79ac0796178cdea4bb5cbf3704b54d4a,
+        type: 3}
+      propertyPath: dashSpeedMultiplier
+      value: 6
+      objectReference: {fileID: 0}
+    - target: {fileID: -5201030127241623248, guid: 79ac0796178cdea4bb5cbf3704b54d4a,
+        type: 3}
+      propertyPath: dashDuration
+      value: 0.2
```

Comment:
Any changes to the player should be made on the prefab, otherwise they could get lost pretty easily. If you select the player in the scene and click the "Overrides" drop down, then you can click "Apply All" to apply all changes to the prefab